doc/classes/AnimatedSprite.xml

```diff
@@ -161,7 +161,7 @@
 			If [code]true[/code] texture is flipped vertically. Default value: [code]false[/code].
 		</member>
 		<member name="frame" type="int" setter="set_frame" getter="get_frame">
-			The current frame index.
+			The index of the animation frame the node displays on screen.
```

Comment:
This is overwritten ("on screen" is unncessary). I liked the first one, but if you want to be more specific, try something like:

    The displayed animation frame's index.